### PR TITLE
Transmit uploaded documents and PDF to Google Drive

### PR DESCRIPTION
**Pull request by bseeger**

### CODE REVIEW

File: src/main/java/org/mdbenefits/app/cli/GoogleDriveClientImpl.java [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+
+    public GoogleDriveClientImpl() throws IOException, GeneralSecurityException {
+        // Build a new authorized API client service.
+        final NetHttpTransport HTTP_TRANSPORT = GoogleNetHttpTransport.newTrustedTransport();
+        service = new Drive.Builder(HTTP_TRANSPORT, JSON_FACTORY, getCredentialsServiceAccount(HTTP_TRANSPORT))
+                .setApplicationName(APPLICATION_NAME)
+                .build();
+    }
+
+    private HttpRequestInitializer getCredentialsServiceAccount(NetHttpTransport httpTransport) throws IOException {
```
[the remark this answers is not in the document]
Comment (bseeger) [reply src/main/java/org/mdbenefits/app/cli/GoogleDriveClientImpl.java:48]: I'll take this out so the alert goes away.

File: src/main/java/org/mdbenefits/app/cli/TransmissionCommands.java [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+                    UserFile file = userFilesForSubmission.get(count);
+
+                    // get the file from S3
+                    CloudFile cloudFile = cloudFileRepository.get(file.getRepositoryPath());
+
+                    String fileName = getUserFileName(confirmationNumber, file, count + 1, userFilesForSubmission.size());
+                    log.info("Uploading file {} of {} for submission with ID: {}.", count + 1, userFilesForSubmission.size(),
+                            submission.getId());
+                    // send to google
+                    googleDriveClient.uploadFile(entryFolderId, fileName, file.getMimeType(), cloudFile.getFileBytes(),
```
Comment (vrajmohan) [inline src/main/java/org/mdbenefits/app/cli/TransmissionCommands.java:130]: For this and the PDF upload, are we not examining `errors` and updating `Transmission` appropriately?

**Review by vrajmohan — APPROVED** [review]
Nice work overall. Some minor questions/comments.

File: src/main/java/org/mdbenefits/app/cli/TransmissionCommands.java [hunk 3]
```diff
@@ -0,0 +1,191 @@
...
+                            file.getFileId().toString(), errors);
+                }
+
+                // upload pdf file
+                googleDriveClient.uploadFile(entryFolderId, pdfFileName, "application/pdf", pdfFileBytes,
```
[the remark this answers is not in the document]
Comment (bseeger) [reply src/main/java/org/mdbenefits/app/cli/TransmissionCommands.java:135]: No reason - I can move it up.  I think what you say makes a lot of sense.  At the very least the PDF will be there and that's the most critical part.  

That reminds me that we have not figured out what to do in the even that not all the info is sendable.

File: src/main/java/org/mdbenefits/app/cli/TransmissionCommands.java [hunk 4]
```diff
@@ -0,0 +1,191 @@
...
+                        confirmationNumber + "_PDF", errors);
+            } catch (IOException e) {
+                log.error("Failed to generate PDF for submission with ID {}: {}.", submission.getId(), e.getMessage());
```
[the remark this answers is not in the document]
Comment (bseeger) [reply src/main/java/org/mdbenefits/app/cli/TransmissionCommands.java:138]: Good point. Moving this.

File: src/main/java/org/mdbenefits/app/cli/TransmissionCommands.java (hunk 2 above)
Comment (bseeger) [reply src/main/java/org/mdbenefits/app/cli/TransmissionCommands.java:130]: The `errors` are updated in the `uploadFiles` method.  

I fixed the gap around PDF generation and PDF upload

**Pull request merged**